terraform/cloud-platform-components/velero.tf

```diff
@@ -0,0 +1,100 @@
+resource "kubernetes_namespace" "velero" {
+  metadata {
+    name = "velero"
+
+    labels = {
+      "component" = "velero"
+    }
+
+    annotations = {
```

Comment:
Should we add "is-production" "true" here?

Reply:
added